OpenRA.Mods.Common/Traits/Player/ParallelProductionQueue.cs

```diff
@@ -0,0 +1,75 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public class ParallelProductionQueueInfo : ProductionQueueInfo
+	{
+		public override object Create(ActorInitializer init) { return new ParallelProductionQueue(init, init.Self.Owner.PlayerActor, this); }
+	}
+
+	public class ParallelProductionQueue : ProductionQueue
+	{
+		public ParallelProductionQueue(ActorInitializer init, Actor playerActor, ParallelProductionQueueInfo info)
+			: base(init, playerActor, info)
+		{
```

Comment:
Style nit: Empty braces tend to go into the line of the function header.

Reply:
This seems still unadressed.

Reply:
So in this case... : base in the next line, afer the ) comes " { }" ? I just wasnt able to find an example with a base block and empty body... It would be done weeks ago, if someone had hinted me an example how you want this to be formatted :)

Reply:
It should be `: base(init, playerActor, info) { }` Here is an example https://github.com/OpenRA/OpenRA/blob/8a6091884127ea2c6706662332ee70097c81885e/OpenRA.Mods.Common/Widgets/Logic/ScreenshotHotkeyLogic.cs#L23